src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -48,6 +52,19 @@ class InstallAppsTask extends AbstractTask {
         if (installProject) {
             installProjectArchive()
         }
+        else if(installDependencies){
+            installDependencies()
```

Comment:
Good catch. It's not declared or implemented anywhere. Interesting that it lets us do that. I'll take it out for now.